src/Microsoft.AspNetCore.Authentication/VirtualSchemeOptions.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Authentication
+{
+    /// <summary>
+    /// Used to redirect authentication methods to another scheme
+    /// </summary>
+    public class VirtualSchemeOptions
+    {
+        public string DefaultTarget { get; set; }
+
+        public string AuthenticateTarget { get; set; }
```

Comment:
Taking a second look at this I wonder if this shouldn't be two different handlers. One that has a list of targets/schemes, and one that only has the selector. Is there a functional advantage to combining them?

Reply:
Just two concept counts instead of one, the meta concept count is authentication policy...so sorta...but there's no reason we couldn't split the behavior in different ways...

Lets keep them together for this iteration, if we don't like how the samples look, we can always split it apart into `AddVirtualScheme/AddSelectorScheme` or something like that in the next iteration